include/vapor/VaporField.h

```diff
@@ -11,6 +11,62 @@
 
 namespace flow
 {
+
+//
+// Helper class: it is used to identify a specific grid.
+//
+class FLOW_API GridKey final
+{
+private:
+    std::string varName; 
+
+    // Buffer definition:
+    // Offset   Data_Type   Information
+    //
+    // 0        uint32_t    currentTS
+    // 4        int32_t     refLevel
+    // 8        int32_t     compLevel
+    // 12       float[3]    extent_min
+    // 24       float[3]    extent_max
+    // 36       uint32_t    variable_dimension
+    // 40       float       DefaultZ
+    // 44       uint8_t     "if this buf should be compared"
+    // Total size: 45 
+    std::array<uint8_t, 45> buf;
```

Comment:
Why is this an anonymous array of bytes rather than a struct?

Reply:
It's just a storage space. In this use case, I only care if the content is the same, but don't care if they're interpreted as floats/ints/doubles etc. As a result, I could use a single `std::strcmp()` function call to compare the entire buffer space, instead of performing a dozen individual comparisons between ints/floats/doubles etc.